packages/jest-validate/README.md

```diff
@@ -0,0 +1,92 @@
+# jest-validate
+
+Generic configuration validation tool that helps you with warnings, errors and deprecation messages as well as showing users examples of correct configuration.
+
+```
+npm install --save-dev jest-validate
+```
+
+## Usage
+
+```js
+import {validate} from 'jest-validate';
+
+validate(
+  config: Object,
+  validConfigSample: Object,
+  deprecatedConfig: ?Object,
+  options: ?Object
+);
```

Comment:
Something like make `validate(config)` work out of box without extra functionality, like providing example valid config option? 
I also thought about `titlePrefix` or `namespace`, so we can prefix all default messages with "🃏" or anything in one place